snap/implicit.go

```diff
@@ -79,3 +85,54 @@ func makeImplicitSlot(snapInfo *Info, ifaceName string) *SlotInfo {
 		Interface: ifaceName,
 	}
 }
+
+// AddImplicitHooks adds hooks from the installed snap's hookdir to the snap info.
+//
+// Existing hooks (i.e. ones defined in the YAML) are not changed; only missing
+// hooks are added.
+func AddImplicitHooks(snapInfo *Info) error {
+	// First of all, check to ensure the hooks directory exists. If it doesn't,
+	// it's not an error-- there's just nothing to do.
+	hooksDir := snapInfo.HooksDir()
+	if _, err := os.Stat(hooksDir); os.IsNotExist(err) {
```

Comment:
I'm not sure I understand... this is just looking for the `meta/hooks` directory. What is too specific?